src/Files.App/Helpers/Win32/Win32Helper.Shell.cs

```diff
@@ -6,23 +6,16 @@
 using Vanara.PInvoke;
 using Vanara.Windows.Shell;
 
-namespace Files.App.Utils.Shell
+namespace Files.App.Helpers
 {
 	/// <summary>
-	/// Provides a utility to manage shell folders.
+	/// Provides static helper for Win32.
 	/// </summary>
-	public class Win32Shell
+	public static partial class Win32Helper
 	{
-		private readonly static ShellFolder _controlPanel;
+		private readonly static ShellFolder _controlPanel = new(Shell32.KNOWNFOLDERID.FOLDERID_ControlPanelFolder);
 
-		private readonly static ShellFolder _controlPanelCategoryView;
-
-		static Win32Shell()
```

Comment:
How can I test these changes?

Reply:
This is the at all same. They were in static constructor and I moved them to each field.